app/components/modals/session-notify-modal.js

```diff
@@ -0,0 +1,8 @@
+import classic from 'ember-classic-decorator';
+import { action } from '@ember/object';
```

Comment:
Codacy found an issue: ['action' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5987150)